util/src/main/java/com/google/cloud/hadoop/util/RetryHttpInitializer.java

```diff
@@ -196,9 +197,6 @@ public boolean handleResponse(HttpRequest request, HttpResponse response, boolea
     }
 
     private void logResponseCode(HttpRequest request, HttpResponse response) {
-      // Incrementing GCS Static Statistics using status code of response.
```

Comment:
is this moved to some other place?

Reply:
Yes, this will be done in the event subscriber.

Reply:
Subscriber is incrementing the metric. HttpResonsecod event is not required. That information is present in the event and it will increment the status code.